client/src/app/containers/Game/models.ts

```diff
@@ -0,0 +1,87 @@
+import { User } from '../../../auth/slice';
+
+export interface Player extends User {
+    color: string;
+}
+
+type Suit = 'Spades' | 'Clubs' | 'Diamonds' | 'Hearts';
+type SuitLetter = 'c' | 'd' | 'h' | 's';
+type ValueLetter =
+    | 'a'
+    | '2'
+    | '3'
+    | '4'
+    | '5'
+    | '6'
+    | '7'
+    | '8'
+    | '9'
+    | '10'
+    | 'j'
+    | 'q'
+    | 'k';
+
+type Value = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13;
+
+// prettier doesn't know about this typescript syntax so we have to disable it here
+// eslint-disable-next-line prettier/prettier
+export type CardName = `${ValueLetter}${SuitLetter}`
+
+export interface Card {
+    name: CardName | 'unknown';
+    suit: Suit;
+    value: Value;
+}
+
+export interface PeggedCard {
+    card: Card;
+    player: string;
```

Comment:
:mag: this is really a `playerID`

Reply:
In my experience, it's not conventional in TypeScript to alias primitive types like you would in go